### DISCUSSION TITLE
Duda con COORDINAR-PEDIDOS y test parte 3-2

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#20] [Tarea 0] [answered]
Tengo una duda respecto al output de la parte 3-2, específicamente hacia el final. El input es ``COORDINAR-PEDIDOS 0 2 3``
Con lo que el output correcto dice que debiese ser esto:

PAQUETES TRANSFERIDOS: 1
    NAVE 0: 12 PEDIDOS
    NAVE 2: 6 PEDIDOS
    
Es decir, hay 1 pedido con planeta 3 en al menos ambas naves. Pero cuando se observa el reporte de naves realizado recientemente se ve que las naves 0 y 2 tienen los siguientes pedidos, respectivamente:
    NAVE 0
        PEDIDO 14428 CON PLANETA 0
        PEDIDO 14857 CON PLANETA 1
        PEDIDO 15847 CON PLANETA 1
        PEDIDO 12547 CON PLANETA 1
        PEDIDO 11857 CON PLANETA 0
        PEDIDO 13441 CON PLANETA 1
        PEDIDO 6474 CON PLANETA 3
        PEDIDO 9416 CON PLANETA 6
        PEDIDO 8453 CON PLANETA 0
        PEDIDO 13708 CON PLANETA 0
        PEDIDO 4777 CON PLANETA 4
        PEDIDO 6939 CON PLANETA 2
    NAVE 2
        PEDIDO 13783 CON PLANETA 5
        PEDIDO 3489 CON PLANETA 0
        PEDIDO 4625 CON PLANETA 0
        PEDIDO 6393 CON PLANETA 0
        PEDIDO 12001 CON PLANETA 1
        PEDIDO 8605 CON PLANETA 2
        
Donde se puede observar que la nave 0 tiene un pedido al planeta 3 y la nave 2 ninguno, por lo que no entiendo cómo se transfiere un pedido... Estoy entendiendo mal algo o el output de ejemplo está mal?

**Comment by PabloNavarroY** [comment, marked as answer]
Era un error. Ahora está arreglado en la última actualización de los tests, aunque se nos olvidó ponerlo en el anuncio.
